core/tests/test_utils.py

```diff
@@ -943,7 +945,7 @@ def save_new_topic(
 
     def save_new_skill(
             self, skill_id, owner_id,
-            description, misconceptions, skill_contents,
+            description=None, misconceptions=None, skill_contents=None,
```

Comment:
isn't a description mandatory for a skill to be saved?

Reply:
Yeah, it was. Done.